src/locale/zh_TW/LC_MESSAGES/woeusb.po

```diff
@@ -1,9 +1,10 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: winusb\n"
-"Report-Msgid-Bugs-To: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/issues>\n"
-"POT-Creation-Date: 2011-07-14 00:08+0100\n"
-"PO-Revision-Date: 2016-07-09 20:51+0800\n"
+"Report-Msgid-Bugs-To: WoeUSB Issue Tracker <https://github.com/slacka/WoeUSB/"
+"issues>\n"
+"POT-Creation-Date: 2011-07-14 00:07+0100\n"
```

Comment:
Should this line be touched?

Reply:
Oh, that is modified by my PO editor, just ignore those.